composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "monero-integrations/monerowhmcs",
+  "description": "A WHMCS Payment Gateway for accepting Monero.",
+  "keywords": [
+    "whmcs",
+    "web host automation platform",
+    "third party gateway module",
+    "gateway module",
+    "third party gateway",
```

Comment:
It's not a common "third party gateway", I would say "self-managed payment gateway", or "payment gateway".